models/ModifyAccountForm.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace app\models;
+
+use yii\base\Model;
+
+/**
+ * ModifyAccountForm is the model behind the modify account form.
+ */
+class ModifyAccountForm extends Model
+{
+    public $firstname;
+    public $lastname;
+    public $email;
+    public $is_validated;
```

Comment:
It's not logical to have is_validated here: it suggests that the admin can edit this "is_validated" value, whereas it's set as true when the user has set his password